gourmet/__main__.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == '__main__':
```

Comment:
I'm not too sure about this file.  
What does it add more to a `--editable` install, for instance?